docs/index.md

```diff
@@ -24,13 +24,13 @@ Welcome to the Danish Foundation Models (DFM) project, a pioneering initiative i
 
 
 
-##  Open-source models on closed-source data
+##  Open-source Development with Privacy-Focused Data Handling
 
-As many of the datasets we use either contain personally sensitive information or fall under copyright they can't be shared publicly. However, we want to share as
-much as possible from the project, while protecting privacy and adhering to copyright law.
-Thus we organize it such that all parts of the project that can be shared and those which
-can't are well-documented using datasheets and training logs.
-Furthermore, during data processing and training, the data is stored on UCloud which follows the highest standards of information security management with a [formal ISO27001 certification](https://docs.cloud.sdu.dk/intro/security.html).
+In our commitment to advancing open-source development, we strongly emphasise the ethical handling of data, particularly when it involves personally sensitive information or material under copyright. This ensures that we share as much as possible while protecting privacy and adhering to copyright laws.
+
+To achieve this, our project is structured to differentiate between data that can be shared openly and that which cannot. This demarcation is documented through detailed datasheets and training logs, hereby ensuring transparency and accountability in our processes.
```

Comment:
> hereby ensuring transparency and accountability in our processes

Since a lot of the data is copyrighted, we should maybe be wary of using "accountability" here. This is exactly what our bosses here are afraid of, that we have to deal with a multimillion lawsuit.

Reply:
Now reads: 

> This demarcation is documented through detailed datasheets and training logs, hereby ensuring transparency in our processes.

Reply:
Thanks btw. exactly the kind of suggestions I wanted.